integration_test/webauthn_test.dart

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022 Yubico.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:yubico_authenticator/core/state.dart';
+
+import 'utils/test_util.dart';
+
+void main() {
+  var binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
+
+  group('FIDO settings', skip: isAndroid, () {
+    appTest('Set FIDO PIN', (WidgetTester tester) async {});
+    appTest('Change FIDO PIN', (WidgetTester tester) async {});
+    appTest('Reset FIDO', (WidgetTester tester) async {});
+  });
```

Comment:
if there are no tests yet, then let's remove the file and add it when we have the test implementations in future PR